src/diff.js

```diff
@@ -27,11 +29,19 @@ export default function diff( a, b, cmp ) {
 		return a === b;
 	};
 
+	const aLength = a.length;
+	const bLength = b.length;
+
+	// Perform `fastDiff` for longer strings/arrays (see #269).
+	if ( Math.min( aLength, bLength ) >= 400 && aLength + bLength >= 1300 || aLength + bLength >= 2000 ) {
```

Comment:
*KISS*. It does not need to be that precise. Also, we should target <50ms because longer delays start becoming visible. So looking at https://github.com/ckeditor/ckeditor5-utils/issues/269#issuecomment-461111923, I'd propose:

```js
aLength > 200 || bLength > 200 || aLength + bLength > 300
```

The worst case scenarios is `a=150,b=150` which should still finish in ~15ms (worth checking) or something around `a=100,b=200`. If we ever need to compare 100 child nodes with other 100 child nodes then we've got a pretty long content and I'd like to see if we need perfect rendering in that case. Perhaps it's fine if we do a bit of DOM overriding in those cases.